templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/fa29b6fd-kura.png",
+                alt="Kura",
+                height="88",
+                width="72",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </a>
+        </li>
+        <li class="p-inline-images__item u-no-margin--left">
+          <a href="https://snapcraft.io/aws-iot-greengrass">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/3a6099ae-greengrass.png",
+                alt="Greengrass",
+                height="88",
+                width="88",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </a>
+        </li>
+        <li class="p-inline-images__item u-no-margin--left">
+          <a href="https://snapcraft.io/azure-iot-edge">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/bc9864b2-azure-iot-hub.png",
+                alt="Azure IOT hub",
+                height="88",
+                width="88",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </a>
+        </li>
+      </ul>          
+      <a href="/internet-of-things/gateways">
+        Learn more&nbsp;&rsaquo;
+      </a>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/d71f9cad-fit-edge-iot.svg",
+          alt="",
+          height="250",
+          width="315",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+  <!-- Industrial micro clouds -->
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/e2bf84de-edge-cluster.svg",
+          alt="",
+          height="250",
+          width="260",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+
+    <div class="col-6">
+      <h2>Industrial micro clouds</h2>
+      <p class="p-heading--four">Cloud-native edge clusters for on premise intelligence</p>
+      <ul>
+        <li>Bring 5G networks and edge clouds to production</li>
+        <li>Deploy AI/ML solutions in industrial settings</li>
+        <li>Get latest versions and security fixes with over-the-air updates</li>
+        <li>Offload IT infrastructure management to our experts</li>
+        <li>Contract long term support services for mission critical software</li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--bottom u-align--left">
```

Comment:
I think `p-inline-images--small` matches the design slightly closer and looks better.